typescript/src/schema/api.ts

```diff
@@ -159,7 +159,11 @@ function getResourceDatabase(
         m?.options?.['.google.api.resource'] as ResourceDescriptor | undefined,
         `file ${fd.name} message ${property}`
       );
+      allResourceDatabase.registerResource(
+        m?.options?.['.google.api.resource'] as ResourceDescriptor | undefined,
+        `file ${fd.name} message ${property}`
+      );
     }
   }
-  return [resourceDatabase, resourceDefinitionDatabase];
+  return [allResourceDatabase, resourceDatabase];
```

Comment:
We need to fix these names :)

Reply:
Sure, was hesitating about a better name, thanks for the suggestion!